src/main.c

```diff
@@ -36,10 +36,6 @@
 #define COLOR_ID        129
 #define SUMMARY_ONLY_ID 130
 
-extern int yydebug;
```

Comment:
I don't think this needs to move to a header.  It's only used here and in the generated parse.{c,h}.